### ISSUE TITLE
Import 'https://deno.land/std@master/examples/welcome.ts' failed: 404 Not Found

### ISSUE THREAD

**Issue by Hardik500** [denoland/deno#7754]
## Issue
Running the programs in the master version from the manual will return a 404 Error

## Steps to reproduce the issue:
1. Go to [https://deno.land/manual](https://deno.land/manual)
2. Select the version as master from the dropdown
3. Click on "2.3 First steps" from the left dropdown or any other section which has a program link with the version
4. Now run any program, let's say `deno run https://deno.land/std@master/examples/welcome.ts` in the terminal
5. We get the following error:
```
Download https://deno.land/std@master/examples/welcome.ts
error: Import 'https://deno.land/std@master/examples/welcome.ts' failed: 404 Not Found
```

## Steps to fix the issue
1. Try removing @master from `deno run https://deno.land/std@master/examples/welcome.ts`
2. Now the code will run the proper output i.e.
```
Welcome to Deno 🦕
```

3. In my opinion we need a specific condition for @master in the dropdown so that it doesn't get added in the URL

I would like to work on this issue if you allow me to.

**Comment by ry** [comment]
> In my opinion we need a specific condition for @master in the dropdown so that it doesn't get added in the URL

Makes sense and your help would be welcome. Probably this change involes https://github.com/denoland/deno_website2

**Comment by Hardik500** [comment]
> Makes sense and your help would be welcome. Probably this change involes https://github.com/denoland/deno_website2

Sorry I didn't realise that there was another repo for the manual.
I guess I should post this issue on [https://github.com/denoland/deno_website2](https://github.com/denoland/deno_website2) @ry and close this one.
